src/docs/development/packages-and-plugins/developing-packages.md

```diff
@@ -223,10 +223,9 @@ environment:
 To create a plugin package, use the `--template=plugin`
 flag with `flutter create`.
 
-Starting from flutter 1.20.0:
-Use the `--platforms` option to specify all the platforms that the plugin
-supports. Available platforms are: android, ios, web, linux, macos and windos.
-If no platforms are specified, a plugin project supports no platforms will be created.
+As of Flutter 1.20.0, Use the `--platforms=` option followed by a comma separated list to 
+specify the platforms that the plugin supports. Available platforms are: android, ios, web, linux, macos, and windows.
```

Comment:
Can we put the platforms in ticks so people know they are exact strings?
`android`, `ios`, `web`, `linux`, `macos`, and `windows`.

Reply:
done